element_interface/intanloader/COPYING

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
+                       Version 3, 29 June 2007

```

Comment:
It might get confusing to mix licenses within a single repo.  Let's brainstorm on alternative options.

Reply:
I'm also concerned about that. Alternatively, we can make the entire module a separate package. But then we would need to add some lines to the COPYING.

Reply:
I like the idea of forking Intan's repo and making modifications there.  That would allow us to pull from the base repository as it is updated, and to contribute back to the base repository if we have fixes.

Reply:
I can move the new code in this PR to https://github.com/datajoint/load-rhs-notebook-python. We will need to change the repos' name, though.

Reply:
Perfect.  How about the following:

1. Keep the `rhs_perchannel_loader` module in the `element-interface` repo, but remove the `rhsutilities` module.  

2. Make the DataJoint fork of the `load-rhs-notebook-python` repo pip installable.  Then we can just import the `rhsutilities` module in `element-interface`.  We will still have the same issue that we have with the `scanreader` package, as it is not on PyPI, but let's deal with that at a later time.

3. Rename the DataJoint fork of `load-rhs-notebook-python`.

Reply:
Sure. Let's try to finish it today.

Reply:
@kabilar I applied all three changes you suggested.